File: android/src/main/java/com/polidea/reactnativeble/utils/Base64Converter.java
package com.polidea.reactnativeble.utils;

import android.util.Base64;

public class Base64Converter {
    public static String encode(byte[] bytes) {
        return Base64.encodeToString(bytes, Base64.NO_WRAP);
    }
    public static byte[] decode(String base64) {
        return Base64.decode(base64, Base64.NO_WRAP);
    }
}

File: android/src/main/java/com/polidea/blenative/utils/Base64Converter.kt
package com.polidea.blenative.utils

import android.util.Base64

object Base64Converter {
    fun encode(bytes: ByteArray): String {
        return Base64.encodeToString(bytes, Base64.NO_WRAP)
    }

    fun decode(base64: String): ByteArray {
        return Base64.decode(base64, Base64.NO_WRAP)
    }
}